docs/pages/cdp/cdp-Installing.md

```diff
@@ -21,7 +21,7 @@ toc: false
 2. Install [Zowe CLI](https://zowe.github.io/docs-site/latest/user-guide/cli-installcli.html). For example, to install using npm:
 
     ```console
-    sudo npm install -g @zowe/cli
+    sudo npm install -g @zowe/cli@latest
```

Comment:
I suggest removing sudo as this is not always required, and in some cases not recommended - see http://michaelb.org/the-right-way-to-do-global-npm-install-without-sudo/